libs/types/src/tokens.rs

```diff
@@ -23,6 +23,16 @@ use sp_runtime::{traits::Get, DispatchError, TokenError};
 
 use crate::{xcm::XcmMetadata, EVMChainId};
 
+/// The type for all Currency ids that our chains handles.
```

Comment:
Nit: Could we enforce a line break to have `#[codec(index =_)]` in the same line? It would also be great, if `NOTE`s always started a new paragraph, either with or without an empty line before.

Moreover, the notes appear to say the same?

```suggestion
/// The type for all Currency ids that our chains handles.
/// Foreign assets gather all the tokens that are native to other chains, such
/// as DOT, AUSD, UDST, etc. 
///
/// NOTE: We MUST never change the `#[codec(index = _)]`
/// below as doing so results in corrupted storage keys. If changing the
/// index value of a variant is mandatory, a storage migration must take place
/// to ensure that the values under an old codec-encoded key are moved to the
/// new key.
```

Reply:
Thanks! apparently I wrote the same NOTE twice, something went wrong there. Fixed